City-of-Helsinki/drupal-helfi
# UHF-36: created paragraph feature

To test:

- `make new`
- enable module `helfi_paragraphs`
- check that new content type 'page' is added
- create new node and add paragraphs of type Text
- try the same with and without enabling `helfi_languages`, and test that asymmetric translations to paragraphs work
